bin/check_archived_drafts.py

```diff
@@ -105,18 +105,28 @@ def main():
         unparsable_modules = []
 
     # Prepare a directory where the missing modules will be copied
-    dst_path = '{}/yangmodels/yang/experimental/ietf-extracted-YANG-modules'.format(missing_modules_directory)
-    if not os.path.isdir(dst_path):
-        os.makedirs(dst_path)
+    dst_path = os.path.join(
+        missing_modules_directory,
+        'yangmodels',
+        'yang',
+        'experimental',
+        'ietf-extracted-YANG-modules',
+    )
```

Comment:
when using `os.path.join()` you don't need to pass each dir as separate argument, you can do this as:
`dst_path = os.path.join(missing_modules_directory, 'yangmodels/yang/experimental/ietf-extracted-YANG-modules')`